projects/Rockchip/devices/RK3399/linux/default/linux.aarch64.conf

```diff
@@ -2977,6 +3306,7 @@ CONFIG_VIDEO_EM28XX_V4L2=m
 CONFIG_VIDEO_EM28XX_ALSA=m
 CONFIG_VIDEO_EM28XX_DVB=m
 CONFIG_VIDEO_EM28XX_RC=m
+# CONFIG_MEDIA_PCI_SUPPORT is not set
```

Comment:
While your at that: could you please set this to "y" and select the PCI(e) DVB drivers as module (I planed to do this for RK3399 anyways)

Reply:
That sounds like a good idea. I'll submit an updated branch shortly.
Note that I can only compile-test that.